company/team/index.md

```diff
@@ -2,7 +2,12 @@
 
 This page contains brief bios of our team. Teammates may also have a personal documentation page in this directory that is named according to their Sourcegraph email address (e.g. you@sourcegraph.com -> you.md).
 
-To add yourself to this page, copy the following template, paste it at the end of this file, and make it about yourself!
+To add yourself to this page:
+1. Accept GitHub's email invite to the Sourcegraph org (you should be a member of [the `Everyone` group](https://github.com/orgs/sourcegraph/teams/everyone) in Sourcegraph's GitHub organization).
+1. Go to the bottom right of this page and click "Edit this page." 
```

Comment:
```suggestion
1. Go to the bottom right of this page and click "Edit this page."  This will direct you to GitHub's file editor user interface. If you would like to learn more about how to edit our handbook, check out [Introduction to git and GitHub](../../handbook/people-ops/onboarding/git_intro.md).
```